src/neo-vm/Types/Struct.cs

```diff
@@ -68,6 +69,11 @@ public override StackItem ConvertTo(StackItemType type)
         }
 
         public override bool Equals(StackItem? other)
+        {
+            return InternalEquals(other, ExecutionEngineLimits.Default.MaxStackSize);
```

Comment:
What if the VM uses different values then `ExecutionEngineLimits.Default`?

Reply:
I think this limit should be a const value not a config value.